### PR TITLE
Uses the “startval” configuration object to set the JSONEditorr initi…

### PR DESCRIPTION
**Pull request by germanbisurgi**
related to issue #10

### CODE REVIEW

File: src/JsonEditorWidget.php [hunk 1]
```diff
@@ -137,7 +137,17 @@ public function run()
         // Prepare client options
         $clientOptions           = $this->clientOptions;
         $clientOptions['schema'] = $this->schema;
-        ArrayHelper::remove($clientOptions, 'startval');
+
+        try {
+            $parsedValue = Json::decode($this->value);
+        } catch (\Exception $e) {
+            $parsedValue = null;
```
Comment (schmunk42) [inline src/JsonEditorWidget.php:144]: Why don't we show an error here?

Comment (germanbisurgi) [reply src/JsonEditorWidget.php:144]: Right. Do we want to throw new exception or just display an error flashcard?

Comment (schmunk42) [reply src/JsonEditorWidget.php:144]: You mean a flash message `Yii::$app->session->addFlash(...)` (error)?
I think so. CC: @handcode WDYT?

We need at least and **always** `Yii::error($e->getMessage(), __METHOD__)` in such cases in the catch block.

Comment (germanbisurgi) [reply src/JsonEditorWidget.php:144]: I added the `Yii::error($e->getMessage(), __METHOD__)`

**Comment by schmunk42** [comment]
LGTM

@handcode @marc7000 Any feedback, veto, etc...?

**Comment by schmunk42** [comment]
:tada:

**Pull request merged**